src/locales/en/messages.json

```diff
@@ -337,5 +337,8 @@
   },
   "OSS_GPT_notAvailable": {
     "message": "OSS-GPT currently is **NOT AVAILABLE** for %v. If you want docs support for the repository, please check [this issue](https://github.com/hypertrons/hypertrons-crx/issues/609) and make a request there :)\n\nSee [all available docs](https://oss.x-lab.info/hypercrx/docsgpt_active_docs.json)\n\nThis chat widget can also be enabled/disabled in the extension options page whenever you want."
+  },
+  "OSS_GPT_errorMessage": {
+    "message": "Sorry, due to cost reasons, the OSS-GPT function is currently out of use. Please follow [HyperCRX](https://github.com/hypertrons/hypertrons-crx) for the latest news."
```

Comment:
-> Sorry, due to cost reasons, the OSS-GPT function is currently **not available**. Please follow [HyperCRX](https://github.com/hypertrons/hypertrons-crx) for the latest news.